internal/endpoints/metrics.go

```diff
@@ -20,10 +20,30 @@ var (
 		Help: "Number of seconds spent waiting on a db response",
 	}, []string{})
 
+	messageProcessElapsed = pa.NewHistogramVec(p.HistogramOpts{
```

Comment:
It looks like messageProcessElapsed, messageProcessError, consumedMessages, and consumeError should really belong in the kafka package.  These metrics appear to only be applicable to the kafka consumer, correct?